src/components/Admin/Toolbar.jsx

```diff
@@ -4,9 +4,34 @@ import Checkbox from "../Checkbox";
 import { HiSearch } from "react-icons/hi";
 import Tag from "./Tag.jsx";
 import { FaDownload, FaTrashAlt } from "react-icons/fa";
+import { CSVLink } from "react-csv";
 import { colors } from "@/data/Tags";
 
-const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
+function convert(input) {
+  if (Array.isArray(input)) {
+    return input.map(convert).join(", ");
+  } else if (typeof input === "boolean") {
+    return input ? "True" : "False";
```

Comment:
what's the default return? like just return input?

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/49844738/833c7893-eb21-416c-b309-398838faac1b)
yes

Reply:
what happens when u just write `return input` in this situation?

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/49844738/85143ae2-e38d-43c9-b838-5b15b26c8db1)
![image](https://github.com/acm-ucr/hackathon-website/assets/49844738/db68e706-903d-4b95-96e7-0221d60c69cf)

Reply:
I mean just for boolean types, if you just return input on boolean types, what is the value?

Reply:
tested it, we don't need it

Reply:
just return input